index.js

```diff
@@ -201,9 +201,11 @@ const MIN_KEYWORD_OCCURRENCES = 2;
  * @param {Array[string]} keywords - An optional array of keywords
  * @param {Array[string]} campaign_types - An optional array of campaign types
  * @param {boolean} load_manually - whether this placement will be loaded manually later
+ * @param {string} force_ad - force the client to show a specific ad
+ * @param {string} force_campaign - force the client to show an ad from a specific campaign
  */
 export class Placement {
-  constructor(publisher, ad_type = "image", target, keywords, campaign_types, load_manually) {
+  constructor(publisher, ad_type = "image", target, keywords, campaign_types, load_manually, force_ad, force_campaign) {
```

Comment:
This constructor is getting a bit unwieldy and I'm thinking accepting an `options` object might be a better approach. Thoughts?

Reply:
Yea, seems reasonable. I don't feel too strongly tho.